extension/encoding/jaegerencodingextension/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package jaegerencodingextension // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/jaegerencodingextension"
+import (
+	"fmt"
+)
+
+type JaegerProtocol string
+
+const (
+	protobuf = JaegerProtocol("protobuf")
```

Comment:
This should be exposed as enum as well

```suggestion
	JaegerProtocolProtobuf JaegerProtocol = "protobuf"
```